core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.component.ts

```diff
@@ -321,6 +329,18 @@ angular.module('oppia').component('topicsAndSkillsDashboardPage', {
         ctrl.filterBoxIsShown = !ctrl.filterBoxIsShown;
       };
 
+      ctrl.getUpperCountForPagination = function() {
+        return (Math.min(((ctrl.pageNumber * ctrl.itemsPerPage) +
```

Comment:
Break after the first '(', and indent subsequent lines by 2.

Reply:
"upper count" sounds weird... Maybe Upper limit? not sure

Reply:
Fixed.